ui/package.json

```diff
@@ -16,8 +16,9 @@
     "test:watch": "jest --watch",
     "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --watch",
     "lint": "yarn run eslint; yarn run tsc",
-    "test:e2e": "CYPRESS_baseUrl=http://localhost:8888 cypress run --browser chrome --headless --reporter junit --reporter-options 'mochaFile=cypress/results/results-[hash].xml'",
-    "test:e2e:headed": "CYPRESS_baseUrl=http://localhost:8888 cypress open --browser chrome",
+    "test:e2e": "nc -z ${OAUTH2_HOSTNAME:=localhost} ${OAUTH2_PORT:=8087} && CYPRESS_oauth2ServerURL=http://${OAUTH2_HOSTNAME}:${OAUTH2_PORT} cypress run --browser chrome --headless --reporter junit --reporter-options 'mochaFile=cypress/results/results-[hash].xml' || echo 'Unable to contact OAuth2 Server. Type \"yarn test:oauth-mock\" to start it.'",
```

Comment:
`nc -z ${OAUTH2_HOSTNAME:=localhost} ${OAUTH2_PORT:=8087}` does the check, but it fails silently, it would be better to have a new script herein that reports what to do and call it instead of just `nc` with `yarn run test:oauth-mock && ...`: 

```js
"test:oauth-mock:check": "(nc -z ${OAUTH2_HOSTNAME:=localhost} ${OAUTH2_PORT:=8087} || echo 'OAuth2 mock server is not running, start it with: yarn run test:oauth-mock') && exit 1",
```

Reply:
I am not sure what exactly do you mean by "fails silently" since there is `echo` at the end of the script, that informs about the OAuth2 server not running and describes how to start it.  Created `test:oauth-mock:check` and used this target in both `test:e2e` and `test:e2e:headed.`